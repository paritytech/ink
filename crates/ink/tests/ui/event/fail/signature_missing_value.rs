#[ink::event(signature_topic)]
pub struct Event {
    pub topic: [u8; 32],
}

fn main() {}
