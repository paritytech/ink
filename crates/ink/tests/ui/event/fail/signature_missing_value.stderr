error: expected a string literal value for `signature_topic` ink! event item configuration argument
 --> tests/ui/event/fail/signature_missing_value.rs:1:14
  |
1 | #[ink::event(signature_topic)]
  |              ^^^^^^^^^^^^^^^
